cmd/protoc-gen-go-grpc/grpc.go

```diff
@@ -170,7 +170,7 @@ func generateFileContent(gen *protogen.Plugin, file *protogen.File, g *protogen.
 
 	g.P("// This is a compile-time assertion to ensure that this generated file")
 	g.P("// is compatible with the grpc package it is being compiled against.")
-	g.P("// Requires gRPC-Go v1.32.0 or later.")
+	g.P("// Requires gRPC-Go v1.32.0 or later.")                     // TODO: Update to latest released gRPC version
 	g.P("const _ = ", grpcPackage.Ident("SupportPackageIsVersion7")) // When changing, update version number above.
```

Comment:
This needs to be updated to `Version8` now.  The comment just lets users discover more easily when `Version8` became available.

Reply:
Done.